pkgs/top-level/aliases.nix

```
@@ -1348,6 +1348,7 @@ mapAliases {
   rtx = mise; # Added 2024-01-05
   runCommandNoCC = runCommand;
   runCommandNoCCLocal = runCommandLocal;
+  run-scaled = throw "run-scaled has been removed from nixpkgs"; # Added 2025-03-17
```

That is not a very helpful error message
```suggestion
  run-scaled = throw "run-scaled has been removed due to being deprecated. Consider using run_scaled from 'xpra' instead"; # Added 2025-03-17
```

---

Done